2023 redesign

depatchedmode opened the pull request:
Full redesign of the website. Has been given the green light by Becky and Bruno.

Lighthouse scores have dropped a tad but they are still fairly good:
<img width="391" alt="image" src="https://user-images.githubusercontent.com/84613835/234367276-c439a3a8-ed15-4abc-a327-150df948a992.png">

Needs an additional accessibility pass, and we have some more functional and aesthetic improvements to work on, but I think those can be tabled for later.

avivash commented on src/_data/projects.js:
```diff
@@ -0,0 +1,86 @@
+require("dotenv").config();
+const Image = require("@11ty/eleventy-img");
+
+var Airtable = require("airtable");
```
not a show stopper, but can we use either `const` or `let` in place of all the `var`s in this file? Can be done after merging

avivash commented on src/_data/team.js:
```diff
@@ -0,0 +1,72 @@
+require("dotenv").config();
+const Image = require("@11ty/eleventy-img");
+
+var Airtable = require("airtable");
```
same note about the `var`s in this file

avivash commented on src/_data/team.js:
```diff
@@ -0,0 +1,72 @@
+        // cache images
+        // TODO: make it so it wasn't written by a designer
+        let teamMember = allTeamMembers[i];
+        let resizedAvatar = await Image(teamMember.sourceAvatarURL, {
```
can be `const`

avivash commented on src/_data/todayProjectIDs.js:
```diff
@@ -0,0 +1,25 @@
+require("dotenv").config();
+
+var Airtable = require("airtable");
+var base = new Airtable({ apiKey: process.env.AIRTABLE_API_KEY }).base(
```
same note about the `var`s

avivash commented on src/_data/tomorrowProjectIDs.js:
```diff
@@ -0,0 +1,25 @@
+require("dotenv").config();
+
+var Airtable = require("airtable");
```
same `var`s note

avivash commented:
@depatchedmode looks like these links are broken(pointing internally instead of externally)

<img width="1481" alt="image" src="https://user-images.githubusercontent.com/1179291/234369864-ddfbc257-a972-457e-81d0-4c9c88d2471d.png">

<img width="1338" alt="image" src="https://user-images.githubusercontent.com/1179291/234370071-cf6ca47a-acfc-4ced-8127-e03cfefbe436.png">

avivash commented:
@depatchedmode looks like some of the links in the nav are _kind of_ broken when viewing the site view IPFS https://ipfs.runfission.com/ipns/fission-2023-redesign-staging.fission.app. I don't think it's a deal breaker though because it looks like they're just hard-coded to point to fission.codes, so it should work once this is deployed to prod

avivash commented:
@depatchedmode looks like there's a merge conflict

depatchedmode commented:
@avivash I've made the var tweaks and updated the content in Airtable to correct the CTA links.

For the merge conflict, it's because I deleted that file and just moved the newsletter into the footer — that's the only place it exists now. Unsure the best way to resolve that; my GitFu is rusty. Could you handle it?

avivash reviewed (approved):
I think we've covered all the mission critical bits for launch, nice work!
